.github/workflows/vercel.yml

```diff
@@ -7,6 +7,10 @@ on:
         description: 'Environment to deploy to. Options are: dev, staging, barn and prod'
         required: true
         type: string
+      app:
+        description: 'Application to deploy. Options are: all, cowswap, explorer, and widget'
```

Comment:
`widget` is not an option anywhere that I can see.

Also, in `deployment.yml`, the options are all upper case, while here all lower case.

Reply:
Thank you! Fixed